code/components/openmetrics/openmetrics.cpp

```diff
@@ -0,0 +1,44 @@
+#include "openmetrics.h"
+
+using namespace std;
+
+/**
+ * create a singe metric from the given input
+ **/
+string createMetric(const string &metricName, const string &help, const string &type, const string &value)
+{
+    return "# HELP " + metricName + " " + help + "\n# TYPE " + metricName + " " + type + "\n" + metricName + " " + value + "\n";
+}
+
+/**
+ * Generate the MetricFamily from all available sequences
+ * @returns the string containing the text wire format of the MetricFamily
+ **/
+string createSequenceMetrics(string prefix, std::vector<NumberPost *> *numbers)
```

Comment:
`numbers` can be a const reference: `const std::vector<NumberPost *>& numbers`. The item type could also be const, but that'd require changes to `getNumbers`'s return type.